docs/requirements_for_rtd.txt

```diff
@@ -41,6 +41,7 @@ python-dateutil~=2.8
 python-memcached~=1.59
 pytz~=2019.3
 pyyaml~=5.1.2
+readthedocs-sphinx-search~=0.1.0
```

Comment:
Ah, now I see that it strips out the `rc1` that I am explicitly passing... this will need to be fixed

Reply:
Ehm... on second look, it seems you managed to comment on an outdated version of the file.
The one I committed yesterday was already fine

Reply:
I made the comments last night, but when I looked this morning, the review wasn't submitted, so that must be the reason.

But what I meant is I think `pip` ignores pre-releases such as release candidates. So normally `pip install readthedocs-sphinx-search` will not install the `rc1`. Now given that this is the one and only release, I am not sure what will happen. Nor what happens when `-r requirements.txt` installing.

Reply:
OK just checked, apparently `pip` makes an exception for pre-releases if it is the only version that exists